scripts/image-names.sh

```diff
@@ -0,0 +1,145 @@
+#!/bin/bash
+
+# This is a quick and dirty script to generate image_vars.src that is compatible with Bash 3.
+# This script can be replaced with for example a Gradle task later.
+
+# TODO: we should be more consistent about naming our images. This script will
+#       help us migrate to get consistency
```

Comment:
I expect to update this functions over time so that they have fewer `case` scenarios, so this script will be much shorter (like the `gradle_image_name` function).